earthpy/spatial.py

```diff
@@ -47,44 +47,51 @@ def extent_to_json(ext_obj):
 
 
 def normalized_diff(b1, b2):
-    """Take two numpy arrays and calculate the normalized difference
-    Math will be calculated (b2-b1) / (b2+b1).
+    """Take two numpy arrays and calculate the normalized difference.
+    Math will be calculated (b1-b2) / (b1+b2).
 
     Parameters
     ----------
     b1, b2 : arrays with the same shape
-        Math will be calculated (b2-b1) / (b2+b1).
+        Math will be calculated (b1-b2) / (b1+b2).
 
     Returns
     ----------
     n_diff : ndarray with the same shape as inputs
-        The element-wise result of (b2-b1) / (b2+b1) with all nan values
-        masked.
+        The element-wise result of (b1-b2) / (b1+b2). Inf values are set
+        to nan. Array returned as masked if result includes nan values.
 
     Examples
     --------
     >>> import numpy as np
     >>> import earthpy.spatial as es
-
-    >>> red_band = np.array([[1, 2, 3, 4, 5],[11,12,13,14,15]])
-    >>> nir_band = np.array([[6, 7, 8, 9, 10],[16,17,18,19,20]])
-
-    >>> # Calculate normalized difference
-    >>> es.normalized_diff(b2=nir_band, b1=red_band)
-    masked_array(
-      data=[[0.7142857142857143, 0.5555555555555556, 0.45454545454545453,
-             0.38461538461538464, 0.3333333333333333],
-            [0.18518518518518517, 0.1724137931034483, 0.16129032258064516,
-             0.15151515151515152, 0.14285714285714285]],
-      mask=[[False, False, False, False, False],
-            [False, False, False, False, False]],
-      fill_value=1e+20)
+    ...
+    ... # Calculate normalized difference vegetation index
+    >>> nir_band = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
+    >>> red_band = np.array([[1, 2, 3, 4, 5], [11, 12, 13, 14, 15]])
+    >>> ndvi = es.normalized_diff(b1 = nir_band, b2 = red_band)
+    ...
+    ... # Calculate normalized burn ratio
+    >>> nir_band = np.array([[8, 10, 13, 17, 15], [18, 20, 22, 23, 25]])
+    >>> swir_band = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
+    >>> nbr = es.normalized_diff(b1 = nir_band, b2 = swir_band)
```

Comment:
doctest output expectations could be provided here too

Reply:
The output of printing ndvi and nbr have been added to normalized_diff in spatial.py.